### PR TITLE
Simplify explorer service node

### PR DESCRIPTION
**Pull request by abrooksv**
## Description
* Pull data from enum so it is in one place
* Rename serviceName to serviceId since that is more accurate

## Testing
Validated Explorer still works

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AbstractActions.kt [hunk 1]
```diff
@@ -16,7 +16,11 @@ import javax.swing.Icon
  *
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AbstractActions.kt]: This whole file just looks like formatting.

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt [hunk 2]
```diff
@@ -26,9 +27,10 @@ import software.aws.toolkits.jetbrains.utils.TaggingResourceType
 import software.aws.toolkits.jetbrains.utils.toHumanReadable
 import software.aws.toolkits.resources.message
 
-class CloudFormationServiceNode(project: Project) : AwsExplorerServiceRootNode(project, message("explorer.node.cloudformation")) {
-    override fun serviceName() = CloudFormationClient.SERVICE_NAME
-
+class CloudFormationServiceNode(
+    project: Project,
+    explorerService: AwsExplorerService
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt]: What's the value of passing this in vs having it be part of the `CloudFormationServiceNode` implementation? Is there a time where `explorerService` would be anything other than `AwsExplorerService. CLOUDFORMATION` ?

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt]: That....is a great question. Got blinders on and didnt see the obvious thing.

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AbstractActions.kt (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AbstractActions.kt]: Ya, will revert

**Pull request merged**